src/test/function_test/test_restore.cpp

```diff
@@ -19,10 +19,8 @@ class restore_test : public testing::Test
     virtual void SetUp() override
     {
         pegasus_root_dir = global_env::instance()._pegasus_root;
-        working_root_dir = global_env::instance()._working_dir;
 
         chdir(pegasus_root_dir.c_str());
-        cluster_name = dsn::utils::filesystem::path_combine(pegasus_root_dir, backup_data_dir);
```

Comment:
It seems that `backup_data_dir` is useless, I think you can remove this variety.